lib/libc/minimal/include/stdlib.h

```diff
@@ -16,13 +16,27 @@
 extern "C" {
 #endif
 
+struct mallinfo2 {
```

Comment:
Can this be put into `mallinfo.h` instead in the common libc directory?

Reply:
I wanted to put this in `malloc.h` as per standard, but I see that malloc is implemented as part of `stdlib`.

Reply:
`malloc.h` is correct (sorry, was mistaken above due to lack of ☕). 

Regardless of where the _declaration_ of `malloc()` is (i.e. in which `.h` file), the declaration for `mallinfo()` should go into `malloc.h`.

The _implementation_ of `malloc()` is in `lib/libc/common/source/stdlib/malloc.c`, so please ensure that you also `#include <mallinfo.h>` in that file too.

Reply:
@krish2718 - can you please address this comment?